config/search.yml

```diff
@@ -4,23 +4,24 @@ site_dir: dist-search
 
 pages:
   - Home: index.md
-  - 'Get started': 'get-started.md'
-  - 'Add search to a map': 'add-search-to-a-map.md'
-  - 'API keys and rate limits': 'api-keys-rate-limits.md'
+  - 'Glossary': 'glossary.md'
   - 'Search': 'search.md'
   - 'Reverse geocoding': 'reverse.md'
   - 'Autocomplete': 'autocomplete.md'
   - 'Place': 'place.md'
+  - 'Tutorial': 'add-search-to-a-map.md'
```

Comment:
`Tutorial` and `Load data from the browser` feel out of place in the order they're in now. Maybe they should go after `HTTP status codes`?